llvm/lib/CodeGen/RegisterCoalescer.cpp

```diff
@@ -1694,12 +1694,19 @@ MachineInstr *RegisterCoalescer::eliminateUndefCopy(MachineInstr *CopyMI) {
   // The source interval may also have been on an undef use, in which case the
   // copy introduced a live value.
   if (((V && V->isPHIDef()) || (!V && !DstLI.liveAt(Idx)))) {
```

Comment:
I know this isn't a part of your changes, but the surplus of parentheses is bugging me.